packages/edit-post/src/components/layout/index.js

```diff
@@ -126,8 +126,8 @@ function Layout( { styles } ) {
 			showIconLabels: select( editPostStore ).isFeatureActive(
 				'showIconLabels'
 			),
-			hasReducedUI: select( editPostStore ).isFeatureActive(
-				'reducedUI'
```

Comment:
I am wondering why we are removing this selector, and if we should still hide the footer if this `reducedUI` setting is `true` ?   Regardless of whether or not the breadcrumbs preference is toggled, reducedUI should still probably hide the footer.  Perhaps we should also only allow/show toggling breadcrumbs preference if we are not in the state of `hasReducdedUI`. 🤔 

All that said, I don't have much knowledge about how/when `hasReducedUI` is used.

Reply:
> All that said, I don't have much knowledge about how/when hasReducedUI is used.

Me neither. I prefer "block breadcrumbs" to be toggled separately. It's more flexible. If "reduced UI" disabled the breadcrumbs as well, then that would lead to confusion IMO.

Reply:
More to the point - it looks like `reducedUI` used to hide the footer.  After these changes it  no longer will.  Im assuming we still want `reducedUI` to have the same behavior regardless of this change?

I guess the possible states could be:
* before this change - `reducedUI` hides footer regardless of breadcrumb
*  combination of `reducedUI` and `! blockBreadcrumb` hides footer.  This would require setting `blockBreadcrumb` false when loading the editor with the `reducedUI` setting to preserve the same experience for people using that setting (i.e. otherwise the footer would randomly start showing up for people running `reducedUI` since it sounds like we are enabling the breadcrumb setting by default? ).
* current state of PR - `blockBreadcrumb` setting itself hides/shows the footer regardless of `reducedUI`

The first seems the most safe since that follows the previous behavior.  The third (current state) seems risky to mess up the intentions of `reducedUI`.  The middle option is more of a compromise between the two.

> I prefer "block breadcrumbs" to be toggled separately. It's more flexible. If enabling "reduced UI" would disable the breadcrumbs as well, then that would lead to confusion IMO.

True.  Im getting at IF we are going with the "before this change" behavior noted above, then going to toggle the block breadcrumb with the footer still hidden wouldn't do anything since its in the reduced UI state.  So my suggestion then would be to hide the breadcrumb toggle while under the state of `reducedUI`, not to necessarily change its setting.

Reply:
> More to the point - it looks like reducedUI used to hide the footer. After these changes it no longer will. Im assuming we still want reducedUI to have the same behavior regardless of this change?

To add to this, given that we also need to show breadcrumbs by default for the time being, this will cause them to appear after the plugin update for users that were not seeing them before (if they had reduced UI enabled)?

Reply:
> To add to this, given that we also need to show breadcrumbs by default for the time being, this will cause them to appear after the plugin update for users that were not seeing them before (if they had reduced UI enabled)?

Correct. Breadcrumbs would be visible for people who had reduced UI enabled.

> The first seems the most safe since that follows the previous behavior. The third (current state) seems risky to mess up the intentions of reducedUI. The middle option is more of a compromise between the two.

That's a good point about making sure reducedUI has the same behavior. I'll push a commit to re-add reducedUI check.